app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -88,8 +88,10 @@
     private static final String CONTRIBUTION_LIST_FRAGMENT_TAG = "ContributionListFragmentTag";
```

Comment:
All the changes in this file seems to be irrelevant. Please revert them back.

Reply:
The changes in the class ContributionsFragment are a result of merge conflicts with the backend-overhaul branch. I was not sure whether these changes are relevant to master. Should I revert these changes?

Reply:
The changes that are present in the current backend_overhaul branch should be left as is (not reverted).

Reply:
These changes are not there in backend-overhaul, your PR is raised on that branch itself and if these changes were from there, these changes would not have shown up. Please re-verify @vanshikaarora.

Reply:
I am sorry Ashish, I thought you were talking about changes relating to merge conflicts i faced in this class. I see that this class generally has formatting changes. 

I'll revert these in my next commit